packages/docs/src/styles/_layout.scss

```diff
@@ -40,6 +40,10 @@ Page layout elements
 .docs-root {
   background-color: $content-background-color;
 
+  // usually this element is _way_ longer than the viewport, but sometimes it isn't.
+  // this ensures that it'll always at least fill the viewport so white page doesn't show through.
```

Comment:
Can we just add a rule to color the whole document background?

```html
html {
  background: @pt-app-background-color;
}
```

Or does that having scoping issues with the `.pt-dark` theme?